src/index.html

```diff
@@ -17,16 +17,29 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <main class="search-bar">
+      <section
+        class="section-search-bar section-search-bar__big"
+        data-qa="big"
+      >
```

Comment:
The use of 'section' here is correct as it is a semantic tag, but the class name 'section-search-bar__big' is not appropriate. According to the checklist, class names should represent the meaning of the content, not the styles or tag names.